src/components/actions/ActionBar.tsx

```diff
@@ -53,6 +54,7 @@ export interface IActionBarProps extends
 export class ActionBar extends React.Component<IActionBarProps, any> {
   static defaultProps: Partial<IActionBarOwnProps> = {
     extraContainerClasses: [],
+    withsSmallActions: false,
```

Comment:
You don't have to define it. `true` or `undefined` is good enough.

Reply:
I think its better to handle a boolean always defined instant of an undefined value. And more, we can see faster the default state for an actionBar rendered